pages/library/_viewId.vue

```diff
@@ -48,8 +47,28 @@ export default Vue.extend({
         sortBy: 'SortName',
         sortOrder: 'Ascending'
       });
+
       this.Items = itemsResponse.data.Items;
     }
+  },
+  methods: {
+    imageLink(id: string) {
+      const url = `${this.$axios.defaults.baseURL}/Items/${id}/Images/Primary`;
+      return url;
+    }
```

Comment:
I couldn't get the imageApi to work so I built the request URL manually.

In the future this should be removed and replaced.

Reply:
https://github.com/jellyfin/jellyfin/pull/4037 should fix this

Reply:
Ahh thanks, I spent too long trying to work out why it wasn't working :p